requirements.txt

```diff
@@ -1,2 +1,3 @@
 lxml
-scipy
\ No newline at end of file
+scipy
+mdtraj
```

Comment:
I'm a bit against growing the dependencies of chem, but a tool for managing the MDSystems could indeed be a good asset. Do you think mdtraj is the correct one? does it also have tools for analyzing the trajectories (RMSD, contacts...)? If so we could build some other protocols / viewers to do that, for the future.

Reply:
I was also thinking we could use it to import other types of MDSystems and also gromacs ones from other places e.g. charmm-gui or some simulations outside scipion